.pre-commit-config.yaml

```diff
@@ -1,25 +1,45 @@
+exclude: >
```

Comment:
would it be possible to add 

```
ci:
  autofix_prs: false
```
  
  As this stops it auto-correcting on commit, and then you having a discrepancy between local and committed versions

Reply:
I think just at the top. As per https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/main/.pre-commit-config.yaml